app/views/govuk_publishing_components/components/docs/input.yml

```diff
@@ -132,6 +132,13 @@ examples:
         text: "What is your name?"
       name: "name"
       heading_size: "l"
+  with_label_as_heading:
```

Comment:
Non-blocker: Whereas in the test it makes sense to check the `heading_level ` in solitude I'm rather worried that presenting this example as separated from the `heading_size` one would make developers chose one or the other instead of using them as a mix. Do you think merging the two examples would bring the code closer to a usual usage?

Reply:
+1 on this - I think that we need to be careful to keep headings looking like headings, even if the heading level and the size are separate in the parameters.

Reply:
Very good shout. Agree that we want headings that look like headings to be headings. I actually can't think of a situation where you wouldn't want this to be the case. I'm gonna try to combine these into a single attribute to force heading levels, maybe with a fallback if a dev explicitly sets it.